frontend/public/extend/kubevirt/module/k8s/vms.js

```diff
@@ -0,0 +1,37 @@
+import * as _ from 'lodash-es';
+
+import { referenceForModel } from '../../../../module/k8s';
+
+export const getResourceKind = (model, name, namespaced, namespace, isList, matchLabels, matchExpressions) => {
```

Comment:
I think this helper obscures the resources being requested more than it helps. I would probably remove it.

Reply:
Hm, the return value of `getResourceKind` seems to be of type `Firehose.propTypes.resources` array item shape.

I'll try to improve the code. cc @mareklibra @rawagner

Reply:
I agree, the long argument list is making its use difficult.
Fine for me to remove it, but please note it's called on multiple places where the flow needs to be easy to maintain.